web/src/OverviewItemView.tsx

```diff
@@ -281,9 +291,164 @@ function buildTooltipText(status: ResourceStatus): string {
+        tooltipText={buildTooltipText(item.buildStatus)}
+        status={item.buildStatus}
+        alertCount={item.buildAlertCount}
+      />
+      <OverviewItemText style={{ margin: "8px 0px" }}>
+        {buildStatusText(item)}
+      </OverviewItemText>
+    </OverviewItemBuildBox>
+  )
+}
+
+type OverviewItemDetailsProps = {
+  item: OverviewItem
+  pathBuilder: PathBuilder
+}
+
+let OverviewItemDetailsRoot = styled.div`
+  display: flex;
+  min-width: 330px;
+  width: calc((100% - 3 * ${SizeUnit(0.75)} - 2 * ${SizeUnit(1)}) / 4);
+  box-sizing: border-box;
+`
+
+let OverviewItemDetailsBox = styled.div`
+  color: ${Color.gray7};
+  background-color: ${Color.gray};
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  width: 100%;
+  border: 1px solid ${Color.grayLighter};
+  border-top: 0;
+  position: relative;
+  font-size: ${FontSize.small};
+  font-family: ${Font.monospace};
+  box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.51);
+  border-radius: 0 0 8px 8px;
+`
+
+let detailsRow = css`
+  outline: none !important;
+  display: flex;
+  background: transparent;
+  cursor: pointer;
+  padding: 0;
+  border: 0;
+  align-items: center;
+  text-decoration: none;
+  font: inherit;
+  color: inherit;
+  margin: 8px 0 8px ${Width.statusIcon + Width.statusIconMarginRight}px;
+  transition: color 300ms ease;
+
+  & .fillStd {
+    fill: ${Color.gray7};
+    transition: fill 300ms ease;
+  }
+  &:hover {
+    color: ${Color.blue};
+  }
+  &:hover .fillStd {
+    fill: ${Color.blue};
+  }
+`
+
+let Endpoint = styled.a`
+  ${detailsRow}
+`
+
+let Copy = styled.button`
+  ${detailsRow}
+`
+
+let ShowDetailsBox = styled(Link)`
+  ${detailsRow}
+`
+
+function displayURL(li: Proto.webviewLink): string {
+  let url = li.url?.replace(/^(http:\/\/)/, "")
+  url = url?.replace(/^(https:\/\/)/, "")
+  url = url?.replace(/^(www\.)/, "")
+  return url || ""
+}
+
+async function copyTextToClipboard(text: string, cb: () => void) {
+  await navigator.clipboard.writeText(text)
+  cb()
+}
+
+export function OverviewItemDetails(props: OverviewItemDetailsProps) {
   let item = props.item
   let link = `/r/${item.name}/overview`
+  let endpoints = item.endpoints.map((ep) => {
+    return (
+      <Endpoint
+        onClick={() => void incr("ui.web.endpoint", { action: "click" })}
+        href={ep.url}
+        // We use ep.url as the target, so that clicking the link re-uses the tab.
+        target={ep.url}
+        key={ep.url}
+      >
+        <LinkSvg />
+        <div style={{ marginLeft: "10px" }}>{ep.name || displayURL(ep)}</div>
+      </Endpoint>
+    )
+  })
+
+  let copy: React.ReactElement | null = null
+  let [showCopySuccess, setShowCopySuccess] = useState(false)
+
+  if (item.podId) {
+    let copyClick = () => {
+      copyTextToClipboard(item.podId, () => {
+        setShowCopySuccess(true)
+
+        setTimeout(() => {
+          setShowCopySuccess(false)
+        }, 5000)
+      })
+    }
+
+    let icon = showCopySuccess ? (
```

Comment:
Aw, nice! We should do more of this. There are still some janky "you copy it yourself 🙃" interactions scattered through Tilt...

Reply:
i can't take credit for this, i copied it from this PR! https://github.com/tilt-dev/tilt/pull/3982